test/controllers/v1/relay_group_membership_controller_test.exs

```diff
@@ -7,13 +7,11 @@ defmodule Cog.V1.RelayGroupMembershipControllerTest do
   @relay_group_create_attrs %{name: "test-relay-group-1"}
 
   setup do
-    # Requests handled by the role controller require this permission
-    required_permission = permission("#{Cog.embedded_bundle}:manage_relays")
-
     # This user will be used to test the normal operation of the controller
     authed_user = user("cog")
     |> with_token
-    |> with_permission(required_permission)
+    |> with_permission("#{Cog.embedded_bundle}:manage_relays")
```

Comment:
It looks like @kevsmith added it to `Cog.Commands.Permissions` when he initially implemented this.